components/tikv_util/src/lib.rs

```diff
@@ -55,6 +57,8 @@ pub mod trace;
 pub mod worker;
 pub mod yatp_pool;
 
+pub use crate::hex::*;
```

Comment:
should remove `pub`?

Reply:
`pub` means that functions in `hex` module will be visible to other crates. If removed, we could not use something like `tikv_util::encode_hex_upper` in other crates (e.g. `log_wrappers`).

Reply:
got it. I misread this line.